### ISSUE TITLE
The load button doesn't allow to load patches

### ISSUE THREAD

**Issue by ycollet** [WeirdConstructor/HexoSynth#6]
I use hexosynth from yesterday master.
I used the standalone version of hexosynth. When I click on the load button, no window shows up to load a patch.

**Comment by ycollet** [comment]
I just read the release note of the latest release:
Preset/Patch management is basically not existent. The "Save" button saves to "init.hxy" and HexoSynth will automatically read that file from the current working directory though.

    Even saving the preset with the DAW project (in case of the VST plugin) is not yet implemented, so don't use it yet for your projects and expect it to work :-)

So, it looks like this button might be not implemented yet :)

**Comment by WeirdConstructor** [comment]
Yes, it's loading currently only `init.hxy` from the current working directory. Save also only saves `init.hxy`. The plugin state/preset should also be saved with the DAW though (VST3 and CLAP plugin).

**Comment by WeirdConstructor** [comment]
> ```
> Even saving the preset with the DAW project (in case of the VST plugin) is not yet implemented, so don't use it yet for your projects and expect it to work :-)
> ```

I've released 0.2.0-alpha-1 yesterday, which actually implements that now. I've fixed some comment in the README.md now.

I'll leave this issue open, as patch management should be improved before a proper release. I can imagine storing patches in some home/application directory later.

**Comment by ycollet** [comment]
OK thanks, I will have a try<
